### PR TITLE
July edits

### PR DESCRIPTION
**Pull request by priyasilverstein**
Stuff added to resources page, clarification in the mailing list guidelines, and adding a new steering com member!

### CODE REVIEW

File: docs/resources.html [hunk 1]
```diff
@@ -67,6 +68,13 @@ <h4 id="resources-for-editors">Resources for editors</h4>
...
+				<h4 id="open-reviews">Open Reviews</h4>
+				<p>It is also possible to make the reviews themselves openly available alongside published manuscripts (with or without the reviewers being identified). This can help to give context to the published article.</p>	
+				<a href="https://" target="_blank">XXX</a>
```
Comment (adam3smith) [inline docs/resources.html:77]: Looks like something missing here?

Comment (priyasilverstein) [reply docs/resources.html:77]: Ah sorry! Was just using that to copy and paste into where I needed it, so you can delete it. Thanks!!

Comment (priyasilverstein) [reply docs/resources.html:77]: (the last line that is)

**Review by adam3smith — COMMENTED** [review]
one quick question above

File: docs/resources.html (hunk 1 above)
Comment (adam3smith) [reply docs/resources.html:77]: will do

**Pull request merged**